python/python/bystro/utils/tests/test_config.py

```diff
@@ -0,0 +1,22 @@
+from unittest.mock import patch
+
+import pytest
+from bystro.utils.config import get_bystro_project_root, get_opensearch_config
+
+
+def test_get_bystro_project_root():
+    bystro_project_root = get_bystro_project_root()
+    assert "bystro" in str(bystro_project_root)
```

Comment:
I don't think this works, because "bystro" will also be in python/python/bystro; so if finding the root fails, we'll still potentially find "bystro". Could we check that os.path.exists(Path.join(bystro_project_root, 'startup.yml')?

Reply:
I think if finding the project root fails (i.e. we fail to find startup.yml) we'll raise FileNotFoundError and so won't make it to the assertion in line 9.  But happy to change the test anyway to make the success condition more explicit.